src/overlays/actors/ovl_Obj_Comb/z_obj_comb.h

```diff
@@ -6,7 +6,14 @@
 
 typedef struct {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x68];
+    /* 0x014C */ ActorFunc actionFunc;
+    /* 0x0150 */ Collider collider;
+    /* 0x0168 */ char unk_168[0x4];
+    /* 0x016C */ ColliderBody* colliderBody;
```

Comment:
this is probably an actual collider body as opposed to a pointer to one, no? then some of the following unk chars would be filled

Reply:
disregard, it is a pointer. though looking in the game, the actual colliderbody starts right after the pointer instance variable